numba/datamodel/testing.py

```diff
@@ -0,0 +1,155 @@
+from __future__ import print_function, absolute_import
```

Comment:
It will be reused and can appear in multiple test files as more Type and DataModel are defined.